### PR TITLE
Add speaker

### PR DESCRIPTION
**Pull request by illia-stv**

### CODE REVIEW

File: src/regulamin.html [hunk 1]
```diff
@@ -31,15 +31,49 @@
 
   <body>
     <script type="text/javascript">
-(function(window, document, dataLayerName, id) {
```
Comment (JKrymarys) [inline src/regulamin.html:34]: Is this change intentional?

**Review by JKrymarys — APPROVED** [review]
Looks good, apart from one change (I left the comment)

**Comment by naugtur** [comment]
Why change the photo of Aleksandra?

File: src/index.html [hunk 2]
```diff
@@ -704,6 +704,36 @@ <h4>
...
+                />
+                <div class="talks-box__info">
+                  <div class="talks-box__header">
+                    <div>
+                      <h3>Jakub Pusiak</h3>
+                      <a
+                        href="https://twitter.com/le_brande"
+                        target="_blank"
+                        title="Twitter profile - Jakub Pusiak"
+                        >#danieljcafonso</a
```
Comment (naugtur) [inline src/index.html:722]: Jakub or Daniel?

File: src/regulamin.html (hunk 1 above)
Comment (naugtur) [reply src/regulamin.html:34]: build autoformats code. something updated with new opinions. This is fine
